ethereum/contracts/governance/Governance.sol

```diff
@@ -164,7 +164,7 @@ contract Governance is IGovernance, Ownable2Step {
     /// @notice Executes the scheduled operation after the delay passed.
     /// @dev Both the owner and security council may execute delayed operations.
     /// @param _operation The operation parameters will be executed with the upgrade.
-    function execute(Operation calldata _operation) external onlyOwnerOrSecurityCouncil {
+    function execute(Operation calldata _operation) external payable onlyOwnerOrSecurityCouncil {
```

Comment:
Should we add test?

Reply:
I think for this function we may skip it, since it would be basically testing that `payable` works